behavior_tests/src/cmake_dpct_helper_sycl_compile/do_test.py

```diff
@@ -0,0 +1,60 @@
+# ====------ do_test.py---------- *- Python -* ----===##
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+#
+# ===----------------------------------------------------------------------===#
+import os
+
+from test_utils import *
+from test_config import CT_TOOL
+
+rel_bin_path = "./build/test_dpct_helper_sycl_compile"
+
+
+def setup_test():
+    change_dir(test_config.current_test)
+    return True
+
+
+def migrate_test():
+    """
+    Builds the project using cmake. The cmake list contains
+    dpct_helper_sycl_compile.
+    """
+    # clean previous migration output
+    if os.path.exists("build"):
+        shutil.rmtree("build")
+
+    # configure and build cmake containing dpct_helper_sycl_compile
+    build_cmd = (
+        "cmake -DCMAKE_CXX_COMPILER=icpx -B build -S . "
+        "&& cmake --build build --parallel"
```

Comment:
```suggestion
        "&& cmake --build build"
```

Fails on Windows with that options.